system/LINK/bin/fix_auth

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# description: ManageIQ fix_auth launcher
+
+cd /var/www/miq/vmdb
+bundle exec ruby tools/fix_auth.rb "$@"
```

Comment:
@kbrock should we capture pwd before this script and return to that directory after?  Is it weird for your directory to change on you?